wordpress-mobile/WordPress-Android
# Update Fastlane and release-toolkit

This PR updates:
-  `Fastlane`  in order to fix an issue with some actions that were not recognized.
- `release-toolkit` in order to fix an issue with the `configure_apply` action.

To test:
- Verify that the CI is green.
- Run `bundle install`.
- Run `bundle exec fastlane run configure_apply` and verify that it finishes without errors. 

**Note:**
This should not be merged after approval because we'll do a proper `release-toolkit` release and update the reference.

PR submission checklist:
